consoles/vnc_base.pm

```diff
@@ -130,32 +130,35 @@ sub type_string {
             "\e" => 'esc'
         };
         $letter = $charmap->{$letter} || $letter;
-        $self->{vnc}->map_and_send_key($letter);
-        $self->{backend}->run_capture_loop($seconds_per_keypress);
+        # 25% is spent hitting the key, 25% releasing it, 50% seaching the next key
```

Comment:
seaching?

Reply:
https://context.reverso.net/%C3%BCbersetzung/englisch-deutsch/seach

Reply:
we "search" for the key? Is it a "keyboard from hell" that dynamically changes the layout after each key press? ;)

Reply:
two finger key hunters reach up to 30wbm!